java/java.lexer/src/org/netbeans/api/java/lexer/JavaTokenId.java

```diff
@@ -133,6 +133,9 @@
     TRUE("true", "literal"),
     FALSE("false", "literal"),
     NULL("null", "literal"),
+    PERMITS("permits", "keyword"),
```

Comment:
These new constants do not seem to be really used, right? I.e. the lexer will never produce a token with this kind?